server/src/services/store/templates/draft-2-command-line-tool.hbs

```diff
@@ -0,0 +1,45 @@
+{
+    "id": "{{ id }}",
+    "class": "CommandLineTool",
+    "label": "{{ label }}",
+    "description": "{{ description }}",
```

Comment:
Not sure how useful interpolating description would be for templates. The user would have to enter it when creating a new file, but descriptions are supposed to be lengthy. 

The way SB handles the label is by asking the user for the name/title/label of the tool and then slugifying the name to create the unique id (in this case that would be the file name)